src/buildfile

```diff
@@ -5,9 +5,9 @@ impl_libs = # Implementation dependencies.
 ./: lib{benchmark} liba{benchmark-main}
 
 pub = [dir_path] ../include
-pub_hdrs = $pub/{$pub_hdrs}
+include($pub)
 
-lib{benchmark}: cxx{* -benchmark_main} hxx{*} $pub/$pub_hdrs
+lib{benchmark}: cxx{* -benchmark_main} hxx{* $pub/benchmark/benchmark} 
```

Comment:
We don’t need to expand the public header multiple times. By placing the header again inside `hxx` we ask `build2` to “glob” for it which is not needed.
Besides by separating `src/` and `include/` we should try to maintain a level of abstraction in general. `src/` does not need to know what the public headers are and it’s best to keep it this way